src/eeg_series.jl

```diff
@@ -147,59 +160,59 @@ function eeg_topoplot_series!(
         data_mean = data
     end
     (q_min, q_max) = extract_colorrange(to_value(data_mean), y)
-    topoplot_attributes = merge(
-        (
-            colorrange = (q_min, q_max),
-            interp_resolution = (128, 128),
-            #contours = (; levels = range(q_min, q_max; length = 7)),
-        ),
-        topoplot_attributes,
-    )
+    topo_attributes = update_axis(topo_attributes; colorrange = (q_min, q_max))
+
     # do the col/row plot
     axlist = []
     if interactive_scatter != nothing
         @assert isa(interactive_scatter, Observable)
     end
     for r = 1:length(select_row)
         for c = 1:length(select_col)
-            ax = Axis(fig[:, :][r, c]; axis_options...)
             df_single =
                 topoplot_subselection(data_mean, col, row, select_col, select_row, r, c)
+            single_y = @lift($df_single[:, y])
+            if isempty(to_value(single_y))
+                break
+            end
+
+            ax = Axis(     #here we loose 30 seconds
```

Comment:
what does loose 30seconds mean?

Reply:
probabaly i meant that creating axis requires 30 ms